src/app/services/enclosure/enclosure.octoprint.service.ts

```diff
@@ -84,6 +85,24 @@ export class EnclosureOctoprintService implements EnclosureService {
       .subscribe();
   }
 
+  setPWM(identifier: number, dutyCycle: number): void {
+    const pwmBody: EnclosurePWMBody = {
+      dutyCycle,
```

Comment:
```suggestion
      /* eslint-disable camelcase */
      duty_cycle: dutyCycle,
```

I think Typescript doesn't automatically convert camelCase to snake_case

Reply:
Indeed, fixed.